### PR TITLE
Added extra explanation to Presets menu

### PR DESCRIPTION
**Pull request by Syed-Ansar**
[Added extra explanation to Presets menu #180 ](https://github.com/publiclab/infragram/issues/180)

-               <p>Choose from these presets to get up and running quickly:</p>
+               <p>Choose from these presets to get up and running quickly. These auto-load common 2. Analysis and 3. Colorize settings so you don't have to manually choose them.</p>

please checkout my pull request

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @Syed-Ansar, can you add the link to the issue in this format `Fixes #0000`? Thank you!

**Comment by Syed-Ansar** [comment]
Hello @TildaDares I don't know how to add because it is my first PR,  can you help me out ?

File: pi/index.html [hunk 1]
```diff
@@ -1,212 +1,243 @@
...
-                         <a class="btn btn-success" id="preset_ndvi_red_color">Colorized</a>
-                         <a class="btn btn-success" onClick="boost('red')">Colorized + boosted (for Pi cameras)</a></td>
-                   </tr>
-                 </table>
-                 <p>Post a research note at <a href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
-
-	    </div>
-            <div class="modal-footer">
-              <a data-dismiss="modal" data-toggle="modal" class="btn">cancel</a>
-            </div>
```
Comment (TildaDares) [inline pi/index.html:68]: @Syed-Ansar Your linter/formatter has added more changes than necessary. You need to disable it and then undo these changes.

**Review by TildaDares — CHANGES_REQUESTED** [review]
